bindata/v3.11.0/openshift-controller-manager/ds.yaml

```diff
@@ -55,8 +57,13 @@ spec:
       - name: serving-cert
         secret:
           secretName: serving-cert
+      - name: proxy-ca-bundles
+        configMap:
+          name: openshift-global-ca
+          items:
+            - key: ca-bundle.crt
+              path: tls-ca-bundle.pem
       nodeSelector:
         node-role.kubernetes.io/master: ""
-      priorityClassName: "system-cluster-critical"
```

Comment:
note the built in yaml validation in GoLand figured out that this was a duplicate setting with line 25 above